### DISCUSSION TITLE
Possible to add a button or link to a BarList list item?

### DISCUSSION THREAD

**Discussion by axiosonlyfwd** [tremorlabs/tremor-npm#926] [Q&A]
Hey Tremor,

I've been using a few of the BarList components and love the styling but was wondering if it was possible to make the text on the List a Button or Link item?

For example, I'd love to add a URL or link a different function for each item being passed into the BarList so that users can click on the text and get redirected to a different page.

It wasn't obvious from the documentation or forum whether this was possible so just figured I'd ask if there's anything obvious I could do.

Thank you!

**Comment by axiosonlyfwd** [comment]
Nvm figured it out!

**Reply by ahmednabik** [reply]
Can you share please? Was wondering how to do this myself today
